tox.ini

```diff
@@ -22,7 +22,9 @@ description = Invoke pylint to lint the code
 deps =
     pylint
     {[testenv]deps}
-commands = pylint everviz  tests
+commands =
+    pylint everviz
+    pylint tests -d protected-access
```

Comment:
I think this is reasonable, generally protected access should be avoid, but for testing I think it´s ok.